blog/2021-06-08-build-an-ecommerce-app-with-temporal-part-2.md

```diff
@@ -202,4 +202,6 @@ err := workflow.ExecuteActivity(ctx, SendAbandonedCartEmail, state.Email).Get(ct
 
 Long-living Workflows in Temporal are excellent for scheduled tasks.
 You can build durable time-based logic, like checking whether the user hasn't modified their shopping cart for a given period of time without using a job queue.
-In the next post, we'll look at patterns for unit testing Temporal Workflows.
+In the [next post](https://docs.temporal.io/blog/build-an-ecommerce-app-with-temporal-part-3-testing/), you'll learn about patterns for unit testing Temporal Workflows.
+
+> Update: You can now [find Part 3 here](https://docs.temporal.io/blog/build-an-ecommerce-app-with-temporal-part-3-testing/)!
```

Comment:
```suggestion
In the [next post](/blog/build-an-ecommerce-app-with-temporal-part-3-testing/), you'll learn about patterns for unit testing Temporal Workflows.

> Update: You can now [find Part 3 here](/blog/build-an-ecommerce-app-with-temporal-part-3-testing/)!
```

Reply:
@djmagee should we use relative links for blog posts going forward? I took a look and most other blog posts use absolute URLs starting with `https://docs.temporal.io/blog` for internal links, so I was following that example.

Reply:
Ya, actually we will want all links in the blog to be the full http path.

This is because the blog will be leaving this repo soon.